Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ExternalActionHelper.kt

```diff
@@ -71,12 +72,18 @@ object ExternalActionHelper {
      */
     fun openUrl(fragment: Fragment, url: String) {
         try {
+            val customTabBuilder = CustomTabsIntent.Builder().build()
+            fragment.context?.let { customTabBuilder.launchUrl(it, Uri.parse(url)) }
+
+            // a fallback to a traditional browser could be implemented, if not already handled by Chrome Custom Tabs
```

Comment:
A fallback is not needed as long any browser is installed. Custom tabs just launches a browser.

Reply:
I removed the commented code